### PR TITLE
COMMERCE-10088 - In headless-admin-commerce-catalog API, Products contains both configuration and productConfiguration

### PR DESCRIPTION
**Pull request by victorcdp**
Related Issue: [COMMERCE-10088](https://issues.liferay.com/browse/COMMERCE-10088)

Simple task to remove the Configuration from Product as it's been deprecated by ProductConfiguration

### CODE REVIEW

**Comment by victorcdp** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-api/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Headless Commerce Admin Catalog API
 Bundle-SymbolicName: com.liferay.headless.commerce.admin.catalog.api
-Bundle-Version: 25.1.2
+Bundle-Version: 26.0.0
```
Comment (brianikim) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-api/bnd.bnd]: You cannot include bnd.bnd changes with buildREST. `gw buildREST` regenerates the REST files. bnd changes are done as part of `gw baseline`. You need a separate commit for this with following message "COMMERCE-10088 Semantic Versioning (gw baseline)

Comment (victorcdp) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-api/bnd.bnd]: can you check if it's alright now?

**Comment by brianikim** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/3361

**Pull request closed without merging**